### PR TITLE
Update TGC Dawnstar

### PR DESCRIPTION
**Pull request by robertgk2017**
Updated my fixer for TGC, added link and meta data.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -22745,12 +22745,22 @@ plugins:
         itm: 26
 
   - name: 'The Great City of Dawnstar.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19491/' ]
-    msg: [ *alreadyInTGC ]
+    url:
```
Comment (sibir-ine) [inline masterlist.yaml:22748]: There's trailing whitespace:

```suggestion
    url:
```

File: masterlist.yaml [hunk 2]
```diff
@@ -22745,12 +22745,22 @@ plugins:
...
 
   - name: 'The Great City of Dawnstar.esp'
-    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/19491/' ]
-    msg: [ *alreadyInTGC ]
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/19491/'
+        name: 'The Great City of Dawnstar SSE Edition'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/66898/'
+        name: 'Rob''s Bug Fixes - TGC Dawnstar'
+    msg:
```
Comment (sibir-ine) [inline masterlist.yaml:22753]:
```suggestion
    msg:
```

File: masterlist.yaml [hunk 3]
```diff
@@ -22745,12 +22745,22 @@ plugins:
...
+        subs: [ '[Rob''s Bug Fixes - TGC Dawnstar](https://www.nexusmods.com/skyrimspecialedition/mods/66898/)' ]
+        condition: 'checksum("The Great City of Dawnstar.esp", BB653BFB)'
     clean:
       - crc: 0xBB653BFB
         util: 'SSEEdit v4.0.3'
-
+      - crc: 0xEC69D3F3
+        util: 'SSEEdit v4.0.4d'
+        
   - name: '(The Great City of |TGCo)Dragon Bridge( - Arthmoor DB Patch)?\.esp'
```
Comment (sibir-ine) [inline masterlist.yaml]:
```suggestion
        util: 'SSEEdit v4.0.4d'

  - name: '(The Great City of |TGCo)Dragon Bridge( - Arthmoor DB Patch)?\.esp'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by robertgk2017** [comment]
gotta love notepad++ auto adding whitespace to every line!

**Review by sibir-ine — APPROVED** [review]
Thanks! By default in NPP, you can press Alt+Shift+S to trim trailing whitespace & save (it's in the Macro menu). You can also reconfigure it to Ctrl+S for convenience.

**Pull request merged**